components/ResidentPage/WorkflowOverview.tsx

```diff
@@ -20,19 +20,29 @@ const WorkflowOverview = ({
     ?.filter((w) => !w.submittedAt)
     .filter((w) => w.id !== mostRecent?.id); // exclude most recent workflow
 
+  if (workflows?.length === 0)
```

Comment:
Not seeing an accompanying test in `./components/ResidentPage/WorkflowOverview.spec.tsx` around this change.